### DISCUSSION TITLE
Filter Snap shot.

### DISCUSSION THREAD

**Discussion by HeavyDeception** [P3pp3rF1y/SophisticatedBackpacks#343] [Feature / Idea Archive] [answered]
Ok, so the filter does not have the slots needed to sort for modded items lol.
Could we have the filter upgrade allow for taking an image of the items in the backpack?
I tried by tag but that doesn't really help as I tend to have a bag for mob drops and those tend to not have a mob drop tag.

**Comment by P3pp3rF1y** [comment, marked as answer]
filter upgrade allows to use what's in backpack as a filter so this is already in the mod

**Comment by HeavyDeception** [comment]
Dose not work with the inspection upgrade.
